Library/Homebrew/download_strategy.rb

```diff
@@ -287,10 +287,8 @@ def fetch
 
       ohai "Downloading #{url}"
 
-      resolved_url, _, url_time = resolve_url_basename_time(url)
-
-      fresh = if cached_location.exist? && url_time
-        url_time <= cached_location.mtime
+      fresh = if cached_location.exist? && @url_time
```

Comment:
Would be good to call `resolved_url_and_basename` explicitly given the results are cached anyway.

Reply:
@kabel Ping on this then I'm happy to merge this 👍

Reply:
Good point. It wouldn't have been resolved yet if it was already cached but potentially stale. I added back `resolved_url`.